generator/dart/packages/google_cloud_common/lib/common.dart

```diff
@@ -22,16 +23,17 @@ abstract class Jsonable {
   Object toJson();
 }
 
-abstract class CloudMessage {}
+abstract class CloudMessage implements Jsonable {}
 
-abstract class CloudEnum {
+abstract class CloudEnum implements Jsonable {
```

Comment:
Don't love the `Jsonable` name, not that I can suggest much better names:  `JsonSerializable` ?

Reply:
Perhaps `JsonEncodable`?